Zilon.Core/Zilon.Core/MapGenerators/StaticObjectFactories/OreDepositFactory.cs

```diff
@@ -10,5 +11,8 @@ public sealed class OreDepositFactory : PropDepositFactoryBase
             IDropResolver dropResolver) : base(toolTags: new[] { "pick-axe" }, dropTableSchemeSid: "ore-deposit", PropContainerPurpose.OreDeposits, schemeService, dropResolver)
         {
         }
+
+        protected override int ExhausingValue { get => 10; }
+        protected override DepositMiningDifficulty DepositMiningDifficulty { get => DepositMiningDifficulty.Moderately; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5398720)